tests/integration/src/test/java/org/mule/test/integration/routing/outbound/MulticasterMixedSyncAsyncTestCase.java

```diff
@@ -37,7 +37,7 @@ public void testMixedMulticast() throws Exception {
     List<Apple> results =
         ((List<InternalMessage>) result.getPayload().getValue()).stream().map(msg -> (Apple) msg.getPayload().getValue())
             .collect(toList());
-    assertEquals(2, results.size());
+    assertEquals(3, results.size());
```

Comment:
This is a change in behaviour in MulticastRouter.  This is deliberate, and MulticastRouter will be going away in 4.0 anyway.